extra/setup_lets_encrypt.sh

```diff
@@ -0,0 +1,24 @@
+#!/bin/bash
+
+set -e
+
+curl -h >/dev/null 2>&1 || { echo >&2 "Curl is needed but I cannot find that. Aborting."; exit 1; }
+openssl -h >/dev/null 2>&1 || { echo >&2 "Openssl is needed but I cannot find that. Aborting."; exit 1; }
+
+PWD="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
+cd $PWD \
+    && git clone https://github.com/lukas2511/letsencrypt.sh.git \
```

Comment:
Okay, I'll remove all `&&` since they are indeed unnecessary.

Compared to Official letsencrypt client, which needs python and other dependent softwares, it's more easily to get used to letsencrypt.sh, which only needs curl and openssl.